Gemfile

```diff
@@ -136,6 +136,9 @@ gem 'linkeddata', '~> 3.0'
 gem 'bootstrap_form', '~> 4.4.0'
 gem 'nokogiri', '~> 1.10.10'
 
+# Citation parser to extract meta data for google scholar
+gem 'anystyle'
```

Comment:
could you lock this a little tighter with `'~> 1.4.1`? that should help in the event of a major version api change.